test/system/170-run-userns.bats

```diff
@@ -140,3 +140,16 @@ EOF
         is "${output}" "Error: keep-id is only supported in rootless mode" "Container should fail to start since keep-id is not supported in rootful mode"
     fi
 }
+
+@test "podman userns=keep-id in a pod" {
+    if is_rootless; then
+        user=$(id -u)
+	run_podman pod create --userns keep-id
+	pid=$output
```

Comment:
Eek, am I the only one who read this as "process id" and got severely confused for a second or two?